packages/shared/src/components/MainLayout.tsx

```diff
@@ -200,7 +199,6 @@ export default function MainLayout({
         )}
         {children}
       </main>
-      <AccountCompletionModals />
```

Comment:
Why did we have to remove this and the file completely? I think there are some things inside of it that are needed.

Reply:
Don't think we need any of those fallbacks anymore in this new flow.
But love to hear if I missed something.

Reply:
I see. One question, does this mean the user will always have complete information after the social registration?

Reply:
Yes if they cancel the modal they will be logged out.
If they refresh the page, they will be logged out.